cpu/sam0_common/periph/i2c.c

```diff
@@ -321,15 +323,17 @@ static inline int _write(SercomI2cm *dev, const uint8_t *data, int length)
     DEBUG("Looping through bytes\n");
     while (tmp_data_length--) {
         /* Check that bus ownership is not lost. */
-        if ((dev->STATUS.reg & SERCOM_I2CM_STATUS_BUSSTATE_Msk) != BUSSTATE_OWNER) {
+        if ((dev->STATUS.reg & SERCOM_I2CM_STATUS_BUSSTATE_Msk) 
+            != BUSSTATE_OWNER) {
             DEBUG("STATUS_ERR_PACKET_COLLISION\n");
             return -EAGAIN;
         }
 
         /* Wait for hardware module to sync */
         while (dev->SYNCBUSY.reg & SERCOM_I2CM_SYNCBUSY_MASK) {}
 
-        DEBUG("Written byte #%i to data reg, now waiting for DR to be empty again\n", buffer_counter);
+        DEBUG("Written byte #%i to data reg, now waiting for DR \
+              to be empty again\n", buffer_counter);
```

Comment:
This is not good. The newline is escaped alright, but then there are 14 spaces instead of one.

Reply:
BTW. The way to solve it is like this

           DEBUG("Written byte #%i to data reg, now waiting for DR"
                 " to be empty again\n", buffer_counter);

Reply:
Will fix